network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -48,6 +60,20 @@ def __init__(self, config, output_config_obj=None):
         super().__init__(output_config_obj)
         self.config = config
 
+    @staticmethod
+    def get_query_output(query_answer, only_explanation=False, add_explanation=False):
+        res = query_answer.numerical_result
+        if only_explanation:
+            return res, query_answer.output_explanation
+        query_output = query_answer.output_result
+        if add_explanation:
+            query_output += query_answer.output_explanation
+        return res, query_output
+
+    @staticmethod
+    def get_query_type():
+        return QueryType.SingleConfigQuery
```

Comment:
Should we also have a default query type for `TwoNetworkConfigsQuery`?

Reply:
True, Fixed 
I chose to put the "ComparisonToBaseConfigQuery" as its default.
the method was removed from the relevant classes